front/components/assistant/conversation/AgentMessage.tsx

```diff
@@ -429,24 +476,52 @@ export function AgentMessage({
     }
 
     // TODO(2024-05-27 flav) Use <ConversationMessage.citations />.
+
+    const chainOfThought = agentMessage.chainOfThoughts.join("");
```

Comment:
For now, we render all the chain of thoughts in one box. It's simpler (no need to detect while streaming that it's a "new" CoT) and rendering multiple CoTs needs some design IMO